### ISSUE TITLE
Sharepoint list and point not updating

### ISSUE THREAD

**Issue by hmathias6** [OfficeDev/microsoft-teams-apps-incentives#15]
Hi I'm using the incentives app atm for a training event and it's gone really well until today. The Your Progress isn't updating and nor is the Sharepoint list with the peoples names. Though it is accepting their codes

**Comment by lkanala** [comment]
Hi @hmathias6 ,  Can you check all Incentives you collected by clicking on "View full details" under "Your progress".Some times the Your Progress data  may not get updated immediately, please try refreshing the page . If the problem still exists can you drop a mail to admin@M365x357260.onmicrosoft.com with subject as Incentives as well as comment here. Thanks.

**Comment by hmathias6** [comment]
Hi I managed to find the problem. I had to increase the Data set from the standard 500 to 2000 as we’d had so many people using it! Thanks for getting back to me. This App has revolutionised our training this last week.

Thanks
Hannah

Hannah Mathias, E-Learning Manager


[Description: Description: Description: C&V COLLEGE LOGO_RGB_M]


Cardiff and Vale College
Dumballs Road,  Cardiff CF10 5FE
Coleg Caerdydd a’r Fro
Heol Dumballs, Caerdydd CF10 5FE
Tel / Ffôn 02920 250250
Mobile 07483 975561
Fax / 02920 250339
www.cardiffandvalecollege.ac.uk<http://www.cardiffandvalecollege.ac.uk/>

[cid:image003.png@01D65376.A087F7C0]
